Library/.rubocop.yml

```diff
@@ -8,7 +8,17 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+Homebrew/FormulaCop:
+  Enabled: false
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
Also, as I said before, this will exclude every Formula in personal/custom Taps and Ruby files that are passed directly to `brew style`.

We need to exclude `HOMEBREW_LIBRARY_PATH` rather than include `Taps/**`.